Update FRS docs to match the latest client API (#7507)
microsoft/FluidFramework: docs/content/docs/deployment/azure-frs.md

  connection: config,
};

const client = new AzureClient(clientProps);
```

The user object can also hold optional additional user details. For example:

```javascript
const userDetails: ICustomUserDetails = {
  email: "xyz@outlook.com",
  address: "Redmond",
};

const config = {
  tenantId: "myTenantId",
  tokenProvider: new AzureFunctionTokenProvider(
    "myAzureFunctionUrl" + "/api/GetAzureToken",
    { userId: "UserId", userName: "Test User", additionalDetails: userDetails }
  ),
  orderer: "https://myOrdererUrl",
  storage: "https://myStorageUrl",
};
```

Your Azure Function will generate the token for the given user that is signed using the tenant's secret key and returned to the client without exposing the secret itself.

## Managing containers

The `AzureClient` API exposes `createContainer` and `getContainer` functions to create and get containers respectively. Both functions take in a _container schema_ that defines the container data model. For the
`getContainer` function, there is an additional parameter for the container `id` of the container you wish to
fetch.

In the container creation scenario, you can use the following pattern:

```javascript
const schema = {

>   name: "my-container",

Reviewer: name was removed from schema, right?

Reviewer 2: yup! It's just `initialObjects` and `dynamicObjectTypes` now

  initialObjects: {
    /* ... */
  },
  dynamicObjectTypes: [
    /*...*/
  ],
};
const azureClient = new AzureClient(clientProps);

> const { fluidContainer, containerServices } = await azureClient.createContainer(

Reviewer: ```suggestion
const { container, services } = await azureClient.createContainer(
```

  schema
);

> const id = await fluidContainer.attach();

Reviewer: ```suggestion
const id = await container.attach();
```

```

The `container.attach()` call is when the container actually becomes connected to the service and is recorded in its blob storage. It returns an `id` which is the unique identifier to this container instance.

Any client that wants to join the same collaborative session needs to call `getContainer` with the same container `id`.

```javascript

> const { fluidContainer, containerServices } = await azureClient.getContainer(

Reviewer: ```suggestion
const { container, services } = await azureClient.getContainer(
```

  id,
  schema
);
```

For the further information on how to start recording logs being emitted by Fluid, see [Telemetry]({{< relref "telemetry.md" >}}).

The container being fetched back will hold the `initialObjects` as defined in the container schema. See [Data modeling]({{< relref "data-modeling.md" >}}) to learn more about how to establish the schema and use the `FluidContainer` object.

## Getting audience details

> Calls to `createContainer` and `getContainer` return two values: a `FluidContainer` -- described above -- and a `containerServices` object.

Reviewer: ```suggestion
Calls to `createContainer` and `getContainer` return two values: a `container` -- described above -- and a `services` object.
```

> The `FluidContainer` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `AzureClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the shared objects within the Fluid container can be reused when moving to using `AzureClient`.

Reviewer: ```suggestion
The `container` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `AzureClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the shared objects within the Fluid container can be reused when moving to using `AzureClient`.
```
